addons/goggles/XEH_postInit.sqf

```diff
@@ -91,9 +92,9 @@ player addEventHandler ["Killed",{
         GVAR(DustHandler) = -1;
     };
 }];
-player addEventHandler ["Fired",{[_this select 0, _this select 1] call FUNC(dustHandler);}];
-player AddEventHandler ["Take",{call FUNC(checkGlasses);}];
-player AddEventHandler ["Put", {call FUNC(checkGlasses);}];
+player addEventHandler ["Fired", DFUNC(dustHandler)];
+player AddEventHandler ["Take", DFUNC(checkGlasses)];
+player AddEventHandler ["Put", DFUNC(checkGlasses)];
```

Comment:
Minor, but this uses extra memory. The difference is that the upper version stores `{call FUNCTION}` while the new version stores a full copy of FUNCTION.